pkg/sortition/chain.go

```diff
@@ -51,6 +51,10 @@ type Chain interface {
 	// or not.
 	IsEligibleForRewards() (bool, error)
 
+	// IsOperatorUnstaking checks if the operator is unstaking. It returns true
+	// if the operator has deauthorized their entire stake, false otherwise.
+	IsOperatorUnstaking() (bool, error)
```

Comment:
Ok got it! However, `sortition.Chain` interface defines chain expectations of the `sortition` package. It's imported by `tbtc.Chain` because `tbtc` package needs the same. In this particular case, `IsOperatorUnstaking` is not needed by `sortition` package so it should be placed in `tbtc` package. That said, I think introducing a new `tbtc.StakingChain` interface makes perfect sense.